dis_snek/models/discord_objects/channel.py

```diff
@@ -187,7 +195,8 @@ def history(
     ):
         """
         Get an async iterator for the history of this channel
-        Args:
+
+        Paramters:
```

Comment:
Suppose to be `Args:` here?

```suggestion
        Args:
```

Reply:
Latest push resolved both of these, it's because it was typo'd